### ISSUE TITLE
Fading in between segmented tracks for both intro and loop

### ISSUE THREAD

**Issue by Birutath** [Flazhik/CybergrindMusicExplorer#26]
So yeah, like the title says it does a fade in which kills totally the thing about segmented tracks, which is also something that doesn't happen with the actual game tracks. Am i missing something?

**Comment by Xioverze** [comment]
yeah the fade in will also be used even if the track is looping, which shouldn't be happening
